rules/linux/persistence_linux_backdoor_user_creation.toml

```diff
@@ -0,0 +1,75 @@
+[metadata]
+creation_date = "2023/03/07"
+integration = ["system"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/03/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the attempt to create a new backdoor user by setting the user's UID to 0. Attackers 
+may alter a user's UID to 0 to escalate privileges or establish persistence on a system or domain.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-system.auth-*"]
```

Comment:
Would be nice to validate if this works with Endgame, and test if it really works with auditbeat